src/ArrayContainer.php

```diff
@@ -32,6 +38,7 @@ public function __construct($data = array())
         } else {
             throw new \InvalidArgumentException('The ArrayContainer requires either an array or an array-like object');
         }
+        $this->delegateLookupContainer = $delegateLookupContainer ?: $this;
```

Comment:
I suppose that is true. I guess that just represents what I would do if I was using it. Looks good then.